MANIFEST.in

```diff
@@ -1,12 +1,10 @@
-include AUTHORS.rst
 include CONTRIBUTING.rst
 include HISTORY.rst
 include LICENSE
 include README.rst
 
-recursive-include tests *
-recursive-exclude * __pycache__
-recursive-exclude * *.py[co]
-
+recursive-include tests *.py *.bat *.sh *.md
 recursive-include docs *.rst conf.py Makefile make.bat
-recursive-include parsec/resources *
+recursive-include parsec/core/gui/forms *.ui
```

Comment:
Nothing about the .sql scripts in `parsec/backend/postgresql` ?

Reply:
:+1: